src/main/scala/com/databricks/spark/xml/parsers/StaxXmlParser.scala

```diff
@@ -103,8 +103,8 @@ private[xml] object StaxXmlParser extends Serializable {
       case FailFastMode =>
         val abbreviatedRecord =
           if (record.length() > 1000) record.substring(0, 1000) + "..." else record
-        throw new IllegalArgumentException(
-          s"Malformed line in FAILFAST mode: ${abbreviatedRecord.replaceAll("\n", "")}", cause)
+        logger.debug("Malformed line:", abbreviatedRecord.replaceAll("\n", ""))
```

Comment:
How about:
- Make this consistent with the DropMalformedMode below, logging the line
- But leave `cause` in the exception rather than log here
- Change both to INFO? makes more sense for DropMalformedMode than WARN, really

I'm hesitant to make it DEBUG as those logs typically aren't on and makes diagnosing something that already happened, maybe can't be easily reproduced, harder.